ansible_collections/serverscom/sc_api/plugins/modules/sc_rbs_volume.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2025, Servers.com
+# GNU General Public License v3.0
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: sc_rbs_volume
+version_added: "1.0.0"
+author: "Aleksandr Chudinov (@chal)"
+short_description: Creates, updates or deletes Remote Block Storage volume.
+description: >
+    Creates, updates or deletes Remote Block Storage volume.
+    Returns information about created or updated volume.
+
+options:
+    endpoint:
+      type: str
+      required: false
+      default: https://api.servers.com/v1
+      description:
+        - API endpoint.
+    token:
+      type: str
+      required: true
+      description:
+        - API token.
+    name:
+      type: str
+      required: false
+      description:
+        - Volume name.
+    volume_id:
+      type: str
+      required: false
+      description:
+        - Volume identifier.
```

Comment:
Mutually exclusive with name (it is in the code, but not in docs). Cannot be specified for state=present.